students/eowyn/backcast.py

```diff
@@ -0,0 +1,252 @@
+import pandas as pd
+import numpy as np
+from tabulate import tabulate
+
+fname = 'M426_Processed_Data.txt'
```

Comment:
You so want to get this in a package sooner than later ;-) Or at least broken out into multiple files all in one dir, etc. (though might as well do the package).

But even before then, this stuff should be in a function at least -- mayabe "main()" (or "startup", or "initilize", or??)